ui/client/components/graph/node-modal/editors/EditableEditor.tsx

```diff
@@ -73,7 +73,7 @@ class EditableEditor extends React.Component<Props, State> {
           values={Editor === FixedValuesEditor ? editor.values(param, values) : []}
           validators={editor.validators(param, errors, fieldName || fieldLabel, this.state.displayRawEditor)}
           components={editor.components(param)}
-          formatter={expressionObj.language === "spel" && !isEmpty(spelFormatters[param.typ.refClazzName]) ?
+          formatter={expressionObj.language === "spel" && !isEmpty(spelFormatters[param?.typ.refClazzName]) ?
```

Comment:
spelFormatters is Record, so please change conndition to:
`expressionObj.language === "spel" && spelFormatters[param?.typ.refClazzName] != null ?`

Reply:
Done